src/suparnaNJune20/ArmstrongNo.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Assignment 20 : 28th Jun 2020
+ * Program 4: WAP to check if a number is Armstrong or not.
+Example:-
+Input:- 153
+Output:-153
+hint:- (1*1*1)+(5*5*5)+(3*3*3)
+1+125+27
+153
+ */
+package suparnaNJune20;
+public class ArmstrongNo {
+	void findarmstron(int input) {
+		int amstrongNum = input, sum = 0;
+		System.out.println("Orignal  number " + input);
```

Comment:
Spellcheck please. And spaces in between 2 words.

Reply:
done changes.